src/cpp/session/modules/SessionCodeTools.R

```diff
@@ -914,37 +914,36 @@
    .Call("rs_hasFileMonitor")
 })
 
+.rs.addFunction("doListIndex", function(routine, term, inDirectory, maxCount)
+{
+   if (!.rs.hasFileMonitor())
+      return(NULL)
+   
+   inDirectory <- suppressWarnings(.rs.normalizePath(inDirectory))
```

Comment:
it looks like we formerly handled a NULL for `inDirectory`, but now throw an error. Are all the NULL cases of `inDirectory` already excluded by `.rs.hasFileMonitor`?